### ISSUE TITLE
[Help] data format for fine-tuning

### ISSUE THREAD

**Issue by MrToy** [THUDM/ChatGLM-6B#124]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

How do I prepare the training data for this model? 
As I know GLM use three different mask tokens: [MASK] for short blank filling, [sMASK] for sentence filling, and [gMASK] for left-to-right generation.
But I don't know what token is used for this model, Could you please provide a case, Thanks.

### Steps To Reproduce

train

### Environment

```markdown
- OS: colab
```

**Comment by archwolf118** [comment]
same question， Thank you！

**Comment by MrToy** [comment]
I saw snippet from model.chat, this means
```
            for i, (old_query, response) in enumerate(history):
                prompt += "[Round {}]\n问：{}\n答：{}\n".format(i, old_query, response)
            prompt += "[Round {}]\n问：{}\n答：".format(len(history), query)
```

I think ```[Round {}]\n问：{}\n答：{}\n```   might be the format
